examples/nginx-centos7/Dockerfile

```diff
@@ -0,0 +1,54 @@
+
+# nginx-centos7
+# Here you can use whatever base image is relevant for your application.
+FROM centos:centos7
+
+# Here you can specify the maintainer for the image that you're building
+MAINTAINER Code Guru <codeguru@example.com>
```

Comment:
Who?  I'd prefer something closer to `MAINTAINER Your Name <your.name@example.com>`.